include/mysql.hpp

```diff
@@ -276,7 +279,7 @@ class mysql {
           [&mp, &it, &column, this](auto &item, auto /*i*/) {
             using W = std::remove_reference_t<decltype(item)>;
             if constexpr (std::is_arithmetic_v<W>) {
-              return;
+              // return; // don't return, need ++column at end.
```

Comment:
we don't need to deal with arithmetic type?

Reply:
the column is blob location, if there is an or more arithmetic field before blob, the location is not correct.

Reply:
So there is no arithmetic type here? we can add a static_assert to make sure no arithmetic here.